repos/spack_repo/builtin/packages/papi/package.py

```diff
@@ -74,6 +74,7 @@ class Papi(AutotoolsPackage, ROCmPackage):
     depends_on("cxx", type="build")
     depends_on("fortran", type="build")
 
+    depends_on("perl", when="@7.1.0:", type="build")
```

Comment:
@G-Ragghianti this dependency was added after failing to build PAPI on a centos7 container that did not have perl installed.
This relates to the fix I pushed in PAPI, which has been merged:
https://github.com/icl-utk-edu/papi/pull/464